handler/istorehandler.go

```diff
@@ -14,11 +14,20 @@ import (
 type IStoreHandler interface {
 	plugin.IPluggable
 
-	// GetTrustAnchorIDs returns an array of trust anchor identifiers used
+	// GetTrustAnchorIDs returns a slice of trust anchor identifiers used
 	// to retrieve the trust anchors associated with this token. The trust anchors may be necessary to validate the
 	// entire token and/or extract its claims (if it is encrypted).
 	GetTrustAnchorIDs(token *proto.AttestationToken) ([]string, error)
 
+	// GetRefValueIDs returns a slice of refrence value identifiers used token
```

Comment:
how about the following text:

// GetRefValueIDs returns the list of reference value identifiers used to retrieve the reference values
//  for an attestation scheme, using the claims extracted from attestation token